src/main/java/com/conveyal/datatools/manager/controllers/api/FeedSourceController.java

```diff
@@ -141,6 +139,11 @@ private static void validate(Request req, FeedSource feedSource) {
         if (feedSource.retrieveProject() == null) {
             validationIssues.add("Valid project ID must be provided.");
         }
+        for (String labelId: feedSource.labels) {
+            if (Persistence.labels.getById(labelId) == null) {
```

Comment:
It's a little inefficient to be making multiple individual queries for a single label like this.

Reply:
Disregard if refactoring as suggested in https://github.com/ibi-group/datatools-server/pull/403#pullrequestreview-714190079.

Reply:
I think it'd be a good idea to make one trip to the database here still by fetching using the `in` clause.

Reply:
Refactored in 4ac2e4cab7a40d0c40960ba15503c2781adef26d